README.md

```diff
@@ -124,6 +124,23 @@ To test your local Shardeum network using MetaMask:
 ## Stopping and Cleaning Up
 To stop the network and clean up resources:
 
+```bash
+shardus start 10
+```
+
+This command will start a Shardeum network of 10 nodes, 1 archive server and 1 monitor server exposed through port number 4000 and 3000 respectively. You can inspect the nodes via the `shardus pm2 list` command.
+
+> **Note:** If one of the services fails to start, it might be that the port is still in use from a previous session.
+
+For usage instructions and available options for the shardus command-line tool, run `shardus --help`
```

Comment:
This seems to be in the wrong spot. This section is `Stopping and Cleaning up` but this new content is about starting a network